cms/templates/base.html

```diff
@@ -118,6 +118,7 @@
       % endif
 
       <div id="page-alert">
+        <%include file="widgets/deprecated-course-key-warning.html" args="context_course=context_course" />
```

Comment:
"Hard-code" it here (in the base template), regardless of the `page_alert` block provided by the inheriting pages, so that this is included on _all_ studio pages, though it will only appear on course-level pages.